saw-central/src/SAWCentral/Builtins.hs

```diff
@@ -1887,13 +1907,14 @@ eval_list t = do
   ts <- io $ traverse (scAt sc n' a' (ttTerm t)) idxs
   return (map (TypedTerm (TypedTermSchema (C.tMono a))) ts)
 
-term_theories :: [String] -> TypedTerm -> TopLevel [String]
+term_theories :: [Text] -> TypedTerm -> TopLevel [Text]
 term_theories unints t = do
   sc <- getSharedContext
-  unintSet <- resolveNames unints
+  unintSet <- resolveNames $ map Text.unpack unints
   hashConsing <- gets SV.rwWhat4HashConsing
   prop <- io (predicateToProp sc Universal (ttTerm t))
-  Prover.what4Theories unintSet hashConsing (propToSequent prop)
+  theories <- Prover.what4Theories unintSet hashConsing (propToSequent prop)
+  return $ map Text.pack theories
```

Comment:
Any reason not to make `what4Theories` return `[Text]` here so that this `map Text.pack` step isn't required? `what4Theories` is defined within `saw-central`, so it seems within the scope of this PR.